src/main/java/com/google/maps/model/OpeningHours.java

```diff
@@ -0,0 +1,58 @@
+package com.google.maps.model;
+
+public class OpeningHours {
+  /**
+   * openNow is a boolean value indicating if the place is open at the current time.
+   */
+  public Boolean openNow;
+
+  static public class Period {
+    static public class OpenClose {
+      public enum DayOfWeek {
```

Comment:
That sounds fine, I wasn't sure what version of Java it was added. If it's Java 8, then definitely ignore it and use this guy.